base/applications/network/netstat/netstat.c

```diff
@@ -482,7 +482,7 @@ VOID ShowTcpTable(VOID)
                 PID[0] = 0;
             }
 
-            ConPrintf(StdOut, L"  %-6s %-22s %-22s %-11s %s\n", L"TCP",
+            ConPrintf(StdOut, L"  %-6s %-22S %-22S %-11S %s\n", L"TCP",
```

Comment:
The `%s` corresponding to TcpState should be lowercase, the one for PID uppercase.